### ISSUE TITLE
Pivot accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1613]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Titles are not announced for the pivots
Expect: When a pivot gets focus, announce the name

2. Doesn't announce when a pivot is selected
Expect: Should say "<pivot name> selected"

**Comment by betrue-final-final** [comment]
#1 does not repro in Narrator, but #2 does.

**Comment by betrue-final-final** [comment]
No longer reproes.
